flow/connectors/postgres/postgres.go

```diff
@@ -42,6 +42,7 @@ type PostgresConnector struct {
 	customTypesMapping     map[uint32]string
 	hushWarnOID            map[uint32]struct{}
 	relationMessageMapping model.RelationMessageMapping
+	pgVersion              *shared.PGVersion
```

Comment:
```suggestion
	pgVersion              shared.PGVersion
```
Can treat `0` as uninitialized